### PR TITLE
Cibw: build more wheels

### PR DESCRIPTION
**Pull request by mayli**
added more wheels via cibw, feel free to squash commit before merge

https://github.com/mayli/fast-bencode/actions/runs/8351699087

### CODE REVIEW

File: .github/workflows/build_whl.yml [hunk 1]
```diff
@@ -5,31 +5,23 @@ on:
...
-      run: |
-        python -m pip install --upgrade pip
-        python -m pip install --upgrade setuptools
-        python -m pip install --upgrade wheel
-        pip install flake8 pytest
-        pip install -r requirements.txt
-    - name: build_whl
-      run: |
-        python setup.py sdist bdist_wheel
+    - uses: actions/setup-python@v3
```
[the remark this answers is not in the document]
Comment (mayli) [reply .github/workflows/build_whl.yml:20]: It's the python version for host/cibw, the actual build will happen inside cibw's containers, which will be isolated from the host version.

**Comment by synodriver** [comment]
Thanks for your contribution! Honestly I'm not very familiar with CI build wheel, I used to use actions to build and publish a release. Anyway, I'll have a try.

**Comment by synodriver** [comment]
Is it Ok to merge it now?

**Comment by mayli** [comment]
> Is it Ok to merge it now?

I am waiting for the https://github.com/mayli/fast-bencode/actions/runs/8352021867
it should be safe to merge, since CIBW is quite popular, and the `upload.yml` could be mirrored to upload wheels to pypi.

**Comment by synodriver** [comment]
> > Is it Ok to merge it now?
> 
> I am waiting for the https://github.com/mayli/fast-bencode/actions/runs/8352021867 it should be safe to merge, since CIBW is quite popular, and the `upload.yml` could be mirrored to upload wheels to pypi.

Fine, I'll release an alpha version for testing later.

**Comment by synodriver** [comment]
Try [this one](https://github.com/synodriver/fast-bencode/actions/runs/8466715500). Pypi is down at this time.

**Pull request merged**